v2/blockstore/doc.go

```diff
@@ -15,4 +15,14 @@
 // instantiated from the same file path using OpenReadOnly.
 // A user may resume reading/writing from files produced by an instance of ReadWrite blockstore. The
 // resumption is attempted automatically, if the path passed to OpenReadWrite exists.
+//
+// Note that the blockstore implementations in this package behave similarly to IPFS IdStore wrapper
+// when given CIDs with multihash.IDENTITY code.
+// More specifically, for CIDs with multhash.IDENTITY code:
+// * blockstore.Has will always return true
+// * blockstore.Get will always succeed, returning the multihash digest of the given CID
+// * blockstore.GetSize will always succeed, returning the multihash digest length of the given CID
+// * blockstore.Put and blockstore.PutMany will always succeed, without writing any bytes onto the backing file.
```

Comment:
how about "will always succeed without performing any operation"? because then it's also clear that it does not affect the index